packages/esm-patient-vitals-app/src/vitals-biometrics-form/vitals-biometrics-form.component.tsx

```diff
@@ -133,98 +238,56 @@ const VitalsAndBiometricsForm: React.FC<DefaultWorkspaceProps> = ({
-      config.concepts.pulseUuid,
-      config.concepts.respiratoryRateUuid,
-      config.concepts.systolicBloodPressureUuid,
-      config.concepts.temperatureUuid,
-      config.concepts.weightUuid,
-    ],
-  );
-
   const savePatientVitalsAndBiometrics = useCallback(
     (data: VitalsBiometricsFormData) => {
       const formData = data;
-      setShowErrorMessage(true);
-      setShowErrorNotification(false);
+      setShowFormSubmissionErrorNotifications(false);
 
       data?.computedBodyMassIndex && delete data.computedBodyMassIndex;
 
-      const allFieldsAreValid = Object.entries(formData)
-        .filter(([, value]) => Boolean(value))
-        .every(([key, value]) => isValueWithinReferenceRange(conceptMetadata, config.concepts[`${key}Uuid`], value));
-
-      if (allFieldsAreValid) {
-        setIsSubmitting(true);
-        setShowErrorMessage(false);
-        const abortController = new AbortController();
+      setIsSubmitting(true);
+      const abortController = new AbortController();
 
-        savePatientVitals(
-          config.vitals.encounterTypeUuid,
-          config.vitals.formUuid,
-          config.concepts,
-          patientUuid,
-          formData,
-          abortController,
-          session?.sessionLocation?.uuid,
-        )
-          .then((response) => {
-            if (response.status === 201) {
-              invalidateCachedVitalsAndBiometrics();
-              closeWorkspaceWithSavedChanges();
-              showSnackbar({
-                isLowContrast: true,
-                kind: 'success',
-                title: t('vitalsAndBiometricsRecorded', 'Vitals and Biometrics saved'),
-                subtitle: t(
-                  'vitalsAndBiometricsNowAvailable',
-                  'They are now visible on the Vitals and Biometrics page',
-                ),
-              });
-            }
-          })
-          .catch((err) => {
-            setIsSubmitting(false);
-            createErrorHandler();
+      savePatientVitals(
+        config.vitals.encounterTypeUuid,
+        config.vitals.formUuid,
+        config.concepts,
+        patientUuid,
+        formData,
+        abortController,
+        session?.sessionLocation?.uuid,
+      )
+        .then((response) => {
+          if (response.status === 201) {
+            invalidateCachedVitalsAndBiometrics();
+            closeWorkspaceWithSavedChanges();
             showSnackbar({
-              title: t('vitalsAndBiometricsSaveError', 'Error saving vitals and biometrics'),
-              kind: 'error',
-              isLowContrast: false,
-              subtitle: t('checkForValidity', 'Some of the values entered are invalid'),
+              isLowContrast: true,
+              kind: 'success',
+              title: t('vitalsAndBiometricsRecorded', 'Vitals and Biometrics saved'),
+              subtitle: t('vitalsAndBiometricsNowAvailable', 'They are now visible on the Vitals and Biometrics page'),
```

Comment:
Generally, I think it's better to treat the title and subtitle as independent in this case, rather than as two sentences one that follows the other.

```suggestion
              subtitle: t('vitalsAndBiometricsNowAvailable', 'The latest results are now visible on the Vitals and Biometrics page'),
```